source/shared/net/references/concordium-client.rst

```diff
@@ -284,6 +284,19 @@ The ``--name`` option selects which account to import and imports it with this
 name. If it's omitted, all accounts in the file are imported under their
 existing names.
 
+Generate account aliases
```

Comment:
Maybe this should called "View" or "Show" account aliases, as I guess they exist already. As the command is called `show-alias`, the title "Show account aliases" might be right